sci-calculators/qalculate-gtk/qalculate-gtk-4.8.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Note: bump with sci-libs/libqalculate and sci-calculators/qalculate-qt!
+
+inherit xdg
+
+DESCRIPTION="Modern multi-purpose calculator"
+HOMEPAGE="https://qalculate.github.io/"
+SRC_URI="https://github.com/Qalculate/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
should be `LICENSE="GPL-2+"`, see: 
https://github.com/Qalculate/qalculate-gtk/blob/master/src/main.cc#L1C1-L10C1